xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -909,14 +909,14 @@ struct AppModel: ModelProtocol {
         text: String
     ) -> Update<AppModel> {
         // Persist any valid value
-        if let validated = state.nicknameFormField.validated {
+        if let validated = Petname.Part(text) {
```

Comment:
We had a possible desync here between the passed `text` and reading from the form field.